### ISSUE TITLE
508-defect-3 [SCREENREADER, SEMANTIC MARKUP]: aria-label SHOULD be removed when it is redundant

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9134]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

> Found in [VSA BAM2 MDOT MVP a11y spot check](https://github.com/department-of-veterans-affairs/va.gov-team/issues/5868#issuecomment-627438157)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

For the link "Edit permanent address", please remove the `aria-label="Edit permanent address"` as it is unnecessary here because it is identical to the link.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to read through the page content without extraneous, duplicated content creating distractions.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov/hearing-aid-batteries-and-accessories/introduction in browser
2. Log in as username: vets.gov.user+1@gmail.com
3. Start screen reading device
4. Read through the page
5. Verify that the link text and aria-label are redundant

## Possible Fixes (optional)

**Current code**
```html
<a class="vads-c-link vads-u-margin-top--1 vads-u-width--auto" 
  aria-label="Edit permanent address" 
  style="min-width: 8rem;">
    Edit permanent address
</a>
```

**Recommended code**
I also moved the inline CSS to a style declaration for re-use and maintainability.

```html
<style>
.review-card--body a.vads-c-link {
  min-width: 8rem; }
</style>
<a class="vads-c-link vads-u-margin-top--1 vads-u-width--auto">
  Edit permanent address
</a>
```

## Screenshots or Trace Logs

![Screen Shot 2020-05-12 at 11 58 01 AM](https://user-images.githubusercontent.com/57469/81717341-6ae26700-9448-11ea-97bc-94c1d6805974.png)

**Comment by k80bowman** [comment]
Closing since this page no longer exists and I could not find this implementation pattern in `vets-website`.
